ledger/acctupdates.go

```diff
@@ -853,11 +886,118 @@ func (au *accountUpdates) accountsInitialize(tx *sql.Tx) (basics.Round, error) {
 			}
 			accountIdx += trieRebuildAccountChunkSize
 		}
+
+		// we've just updated the markle trie, update the hashRound to reflect that.
+		err = updateAccountsRound(tx, rnd, rnd)
+		if err != nil {
+			return 0, fmt.Errorf("accountsInitialize was unable to update the account round to %d: %v", rnd, err)
+		}
 	}
 	au.balancesTrie = trie
 	return rnd, nil
 }
 
+// upgradeDatabaseSchema0 upgrades the database schema from version 0 to version 1
```

Comment:
I think we need to put summary of migration as a comment

Reply:
will do in a subsequent PR.